src/Sulu/Bundle/SearchBundle/Search/Configuration/IndexConfigurationProvider.php

```diff
@@ -56,6 +71,8 @@ public function getIndexConfigurations()
      */
     public function getIndexConfiguration($name)
     {
+        $this->indexConfigurations = $this->getIndexConfigurations();
+
```

Comment:
we should use a local variable here:

```php
        $indexConfigurations = $this->getIndexConfigurations();

        if (!\array_key_exists($name, $indexConfigurations)) {
            return;
        }

        return $indexConfigurations[$name];
```

Else the static code analyzer will fail as it thinks $this->indexConfigurations can be null and so will throw a error on array_key_exists.